src/Windows/Avalonia.Win32/WindowImpl.AppWndProc.cs

```diff
@@ -171,9 +171,12 @@ protected virtual unsafe IntPtr AppWndProc(IntPtr hWnd, uint msg, IntPtr wParam,
                                 WindowsMessage.WM_XBUTTONDOWN =>
                                 HighWord(ToInt32(wParam)) == 1 ?
                                     RawPointerEventType.XButton1Down :
-                                    RawPointerEventType.XButton2Down
+                                    RawPointerEventType.XButton2Down,
+                                _ => throw new AvaloniaInternalException(
```

Comment:
We should ignore the event instead of throwing from WndProc

Reply:
Here the `case`s in the switch statement must be the same as those in the enclosing `case`: hitting the `_` case means a logic error, so I think it makes sense to throw here.